### ISSUE TITLE
Could this plugin be updated for PHP7 API 2.0

### ISSUE THREAD

**Issue by NML32** [musaichi/CountryBlocker#7]
Could this plugin be updated to work on API 2.0 PHP7 0.14?

**Comment by musaichi** [comment]
Currently I couldn't sorry, but I'm going to update this for 0.14.0 ASAP. I'm really glad if you wait a bit, thank you.

**Comment by NML32** [comment]
Thanks

**Comment by NML32** [comment]
Any updates coming soon so the plugin will work on PHP7 0.14 mcpe?

**Comment by musaichi** [comment]
Yes,But I dont have much time.I'm going to update that this month.Please forgive me.

**Comment by musaichi** [comment]
And I'm sorry for late reply.

**Comment by NML32** [comment]
Thanks, I can wait.
